.github/workflows/flutter-signed-apk.yml

```
@@ -0,0 +1,144 @@
+      - name: Generate google-services.json
+        shell: bash
+        run: |
+          mkdir -p android/app
+          printf '%s\n' "${{ secrets.GOOGLE_SERVICES_JSON }}" \
+            > android/app/google-services.json
+      
+      - name: Show google-services.json head
+        if: ${{ always() }}
+        run: |
+             head -n5 android/app/google-services.json
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)